test/behavior/src.zig

```diff
@@ -3,9 +3,10 @@ const builtin = @import("builtin");
 const expect = std.testing.expect;
 
 test "@src" {
-    if (builtin.zig_backend != .stage1) return error.SkipZigTest;
+    // TODO why is this failing on stage1?
```

Comment:
It's because you changed the line numbers.

Reply:
Mhm, makes sense. I'll push a fix later tomorrow morning then unless you beat me to it. Thanks for pointing this one out Vexu!